src/mocking/mockApi.ts

```diff
@@ -1,78 +1,70 @@
 import MockAdapter from 'axios-mock-adapter';
```

Comment:
Forenkla denne fila litt ved å for eksempel flytte hjelpemetoder inn i en egen mockUtils fil